src/middleware/statement.validation.middleware.ts

```diff
@@ -35,6 +35,11 @@ const validateIdentifiedBOsStatement = (appData: ApplicationData, errorList: str
     return false;
   }
 
+  if (hasNotAddedOrCeasedBos(appData) && appData.update?.[RegistrableBeneficialOwnerKey] === 1) {
```

Comment:
Correct me if I'm wrong, but I think there are two issues here:

1. this method is for checking the model based on the statement stored in `appData[BeneficialOwnersStatementKey]` but this new check is checking against `appData.update[RegistrableBeneficialOwnerKey]`. 
2. the error message `ACTIVE_REGISTRABLE_BO` that's added on line 44 won't be considered if this condition is met, due to the early return statement on line 40.

I _think_ this should be pulled out into its own method and `&&` on line 52.

Reply:
Fixed to separate this out